shared/templates/sysctl/oval.template

```diff
@@ -190,6 +192,13 @@
     {{{ state_static_sysctld("usr_local_lib_sysctld") }}}
   </ind:textfilecontent54_test>
 
+  <ind:textfilecontent54_test id="test_{{{ rule_id }}}_static_lib_sysctld" version="1"
+                          check_existence="any_exist"
+                          check="all"
+                          comment="{{{ SYSCTLVAR }}} static configuration in /lib/sysctl.d/*.conf" state_operator="OR">
+    {{{ state_static_sysctld("lib_sysctld") }}}
+  </ind:textfilecontent54_test>
+
```

Comment:
The same for the `test`, the `object`, and the `state`, could you move them into the conditional in line  179 so that they are no included in RHEL, OL and RHCOS products?

Thanks.